python/Ganga/GPIDev/Adapters/IBackend.py

```diff
@@ -475,10 +472,6 @@ def master_updateMonitoringInformation(jobs):
                     temp_block = []
 
                 for this_block in monitorable_blocks:
-
-                    if monitoring_component and not monitoring_component.isEnabled(False) or not monitoring_component:
```

Comment:
hmmm, This should work, but I was also hoping to allow for the monitoring to be shutdown and restarted quickly during runtime.
If this isn't here the disableMonitoring for instance doesn't take effect until after all of the jobs here have been looped over. (I think)

I think passing a special argument to run without the monitoring here may be a good idea.
Having this only bail out on shutdown risks having the disableMonitoring command becoming slow. I've not seen it auto-called in a long time but it's called in case of a repo error such as the $HOME partition becoming full and write errors occuring.